app/views/admin/_left_hand_navigation.haml

```diff
@@ -3,7 +3,7 @@
 - links_manage = links[2..5]
 - links_configure = links[6..21]
 
-= render :partial => "admin/uservoice_widget"
+= render :partial => "admin/uservoice_widget" if APP_CONFIG.uservoice_widget_url.present?
```

Comment:
hmm.. @sktoiva 
There are 23 files which have
`
render :partial => "admin/left_hand_navigation"
`
and some locals after that.
So pretty many places to put that uservoice specific stuff.. But I guess that doesn't change the principle and I should still put it everywhere?

And is there something else were doing wrong here.. Feels wrong to copy that everywhere..

Actually the thing we're probably doing wrong is to render the left hand navi separately for each page and not rendering that first and then the page contents..